src/home/home.js

```diff
@@ -0,0 +1,29 @@
+import './homeStyles.css';
+import homeImage from '../images/home.jpg';
+
+const HomePage = (() => {
+  const createHomePageMarkUp = () => `
+  <div id="homeId" class="home page">
+  <img id="homeImage" class="home__image"/>
+  <div class="home__info">
+    <h2 class="home__title">Restaurant Page</h2>
+    <button id="btn-home-menu" class="home__button">START EATING</button>
+  </div>
+</div>
```

Comment:
We need to use `document.createElement` and `document.appendChild` to appends HTML elements to the `<div>` with `id=content`

![image](https://user-images.githubusercontent.com/31897434/89599791-67e4f000-d858-11ea-97fe-976b47c8b29c.png)

https://www.theodinproject.com/courses/javascript/lessons/restaurant-page